.expeditor/verify.pipeline.yml

```diff
@@ -708,6 +708,80 @@ steps:
       automatic:
         limit: 1
 
+#######################################################################
+# Things that have no tests but should be built to make sure they
+# still build. - Linux Kernel 2
+#######################################################################
+
+  - label: "[build] :linux: :two: build hab"
+    command:
+      - .expeditor/scripts/verify/build_package.sh components/hab
```

Comment:
This script doesn't appear to understand `BUILD_PKG_TARGET` and will use the `x86_64-linux` hab cli to build.

There's probably a few different ways we can slice this, perhaps easiest would be to run a script before build_package that curl-bashes the kernel2 cli? 

```
command:
  - .expeditor/scripts/verify/install_kernel2_cli.sh
  - .expeditor/scripts/verify/build_package.sh components/hab
```

Reply:
:sob:

Reply:
Once more, with feeling: https://github.com/habitat-sh/habitat/pull/7706/commits/a7455f6728146998e33eb35f19904d9b3b847c17